src/ngraph_mark_for_clustering.cc

```diff
@@ -272,17 +272,8 @@ Status MarkForClustering(Graph* graph) {
       confirmation_functions["FloorDiv"] = SimpleConfirmationFunction();
       confirmation_functions["FloorMod"] = SimpleConfirmationFunction();
       confirmation_functions["FusedBatchNorm"] = SimpleConfirmationFunction();
-      confirmation_functions["FusedBatchNormGrad"] = [](Node* n, bool* result) {
-        // Reject if "is_training" is set false.
-        bool tf_is_training;
-        TF_RETURN_IF_ERROR(
-            GetNodeAttr(n->attrs(), "is_training", &tf_is_training));
-        if (!tf_is_training) {
-          *result = false;
-          return Status::OK();
-        }
-        SimpleConfirmationFunction()(n, result);
-      };
```

Comment:
Now I understand why it was causing the segfault. For the `is_training` case, it did not return `true` in confirmation function because I coded `SimpleConfirmationFunction()(n, result);` instead of `return SimpleConfirmationFunction()(n, result);`. Fixed that bug in the updated code.